packages/contract/src/Account.sol

```diff
@@ -171,7 +153,7 @@ contract Account is BaseAccount, UUPSUpgradeable, Initializable {
     function withdrawDepositTo(
         address payable withdrawAddress,
         uint256 amount
-    ) public onlyOwner {
+    ) public onlySelf {
```

Comment:
I'm not sure if we need deposits

If we use a paymaster
- Non-upgradeable, vetted paymaster contract
- Unlimited approve the paymaster on account creation
- All ops subsequently paid by the paymaster

...do we still need this mechanism for depositing to the entrypoint?

(We don't have to address on this PR.)

Reply:
One edge case that we should think about is if the underlying AA spec gets breaking upgrades and we need to upgrade in sync of those. For instance, the latest upgrade removed the nonce handling from the account itself to the entry point code, (I believe) breaking old deployed accounts. That's mostly why I think having a withdraw is helpful, we can "upgrade" accounts by making a new contract + withdrawing deposit to it if necessary.

Reply:
It's low cost to keep here anyway for now so will punt this delta.